GOVERNANCE.md

```diff
@@ -1,43 +1,102 @@
-# in-toto Governance
-This document covers the project's governance and committer process.  The
-project consists of the in-toto
-[specification](https://github.com/in-toto/docs), multiple implementations, and
-the [attestation framework](https://github.com/in-toto/attestation).
-
-## Maintainership and Consensus Builder
-
-The in-toto project is maintained by the people indicated in
-[MAINTAINERS.txt](MAINTAINERS.txt). Some in-toto sub-projects and
-implementations have maintainers unique to that effort. They are listed in
-similar MAINTAINERS.txt files in the corresponding repositories. In general, a
-maintainer is expected to (1) submit and review GitHub pull requests and
-(2) open issues or
-[submit vulnerability reports](https://github.com/in-toto/in-toto#security-issues-and-bugs).
-A maintainer has the authority to approve or reject pull requests submitted by
-contributors. If a sub-project or implementation has specific responsibilities
-for its maintainers, they are listed in that project's MAINTAINERS.txt file.
-The maintainers of the in-toto specification, listed in this repository's
-[MAINTAINERS.txt](MAINTAINERS.txt) file, can interact with the CNCF on behalf
-of the in-toto project.
-
-The project's Consensus Builder (CB), who is also granted all maintainer 
-privileges and responsibilities, is Santiago Torres-Arias 
-<santiagotorres@purdue.edu, @SantiagoTorres>.  The CB reevaluates the status
-of every maintainer of in-toto's specifications, implementations, and other
-sub-projects annually.
-
-## Changes in maintainership
-
-Active contributors may be offered or request to be granted maintainer status.
-This requires approval from the CB and is done in consultation with the
-current maintainers.
-
-Maintainers may be moved to emeritus status.  This is done at the discretion of 
-the CB, in consultation with the project maintainers.  Emeritus maintainers are 
-listed in the MAINTAINERS.txt file as acknowledgment for their prior service to 
-the project, but no longer have code review or other maintainer privileges for 
-the project.
-
-## Changes in governance
-The CB supervises changes in governance, in consultation with project
-maintainers.
+The in-toto Project succeeds because of an open, inclusive, and respectful community. Ideas and contributions are accepted based on their technical merit and alignment with project objectives, scope, and design principles. This document lists a non-exclusive set of guidelines to help ensure fairness and transparency in managing the in-toto Project.
+
+## Code of Conduct
+
+The in-toto community abides by the Cloud Native Computing Foundation's [code of conduct](/CODE-OF-CONDUCT.md). An excerpt follows:
```

Comment:
This points to https://github.com/in-toto/community/blob/main/CODE-OF-CONDUCT.md which is the exact same text and in turn points to the CNCF document. We should either get rid of this section or the file and point to CNCF CoC directly.